.github/autoissuelabeler.yml

```diff
@@ -16,6 +20,10 @@
 - labels: [area/maven]
   title: "(?i).*maven.*"
   notify: [quarkusio/devtools]
+- labels: [area/hibernate-reactive, area/persistence]
+  title: "(?i).*hibernate reactive.*"
+  description: "(?i).*hibernate reactive.*"
+  notify: [aguibert,gavinking,Sanne]
```

Comment:
Should we notify Davide too?

Reply:
We could add him later, didn't want to add too many people here.